tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/FunctionEmitter.scala

```diff
@@ -957,12 +964,11 @@ private[emitter] class FunctionEmitter(jsGen: JSGen) {
 
               case _ =>
                 val temp = newSyntheticVar()
-                val newEnv = env.withDef(temp, arg.tpe, false)
-                innerEnv = newEnv
                 val computeTemp = pushLhsInto(
                     Lhs.VarDef(temp, arg.tpe, mutable = false), arg,
                     Set.empty)
                 computeTemp +=: extractedStatements
+                innerEnv = innerEnv.withDef(temp, arg.tpe, false)
```

Comment:
Write `mutable = false`? Also a couple of occurrences below.

Reply:
I cannot. This is an *overload* pimped through an implicit class, and apparently that prevents scalac from understanding the named param as such.

Reply:
Blargh :P Ok